app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -159,11 +172,11 @@
   <string name="number_error_starting_with_floating_point">Comece sua resposta com um número (por exemplo, \"0\" em 0,5)</string>
   <string name="number_error_invalid_format">Por favor, insira um número válido.</string>
   <string name="number_error_larger_than_fifteen_characters">A resposta pode conter no máximo 15 dígitos (0–9) ou símbolos (. ou -).</string>
-  <string name="ratio_error_invalid_chars">Escreva uma proporção que consista em dígitos separados por dois pontos (por exemplo, 1:2 ou 1:2:3).</string>
-  <string name="ratio_error_invalid_format">Insira uma proporção válida (por exemplo, 1:2 ou 1:2:3).</string>
+  <string name="ratio_error_invalid_chars">\n    Escreva uma proporção que consista em dígitos separados por dois pontos (por exemplo, 1:2 ou 1:2:3).\n  \\</string>
+  <string name="ratio_error_invalid_format">\n    Insira uma proporção válida (por exemplo, 1:2 ou 1:2:3).\n  \\</string>
   <string name="ratio_error_invalid_colons">Sua resposta tem dois dois-pontos (:) do lado um do outro.</string>
-  <string name="ratio_error_invalid_size">O número de termos não é igual ao exigido.</string>
-  <string name="ratio_error_includes_zero">Proporções não podem ter 0 como elemento.</string>
+  <string name="ratio_error_invalid_size">\n    O número de termos não é igual ao exigido.\n  \\</string>
+  <string name="ratio_error_includes_zero">\n    Proporções não podem ter 0 como elemento.\n  \\</string>
```

Comment:
Invalid newlinesand escape characters

Reply:
I just fixed these directly on translatewiki.

One note: it seems that the English strings might be causing confusion because there's a newline after the opening tag and below the closing tag ([ref](https://github.com/oppia/oppia-android/blob/develop/app/src/main/res/values/strings.xml#L167
)). This results in them looking like this in translatewiki (notice the indent at the start and the line break at the end):

![Screenshot from 2023-09-29 19-52-15](https://github.com/oppia/oppia-android/assets/10575562/7ff0edfc-df3f-4e56-a378-937d996f9935)

@adhiamboperes Would it be possible to update the original file to have each `<string></string>` tag in a single line perhaps (and maybe add a lint check to enforce this)?

Reply:
@seanlip, I checked, we can update all these string values to be on one line. Plurals and quantities are however usually formatted as multiline in the strings files, but each  of their `<item>` tags should also  be on one line. Great catch!